packages/skygear-core/lib/database.js

```diff
@@ -295,20 +314,27 @@ export class Database {
    * The delete will be performed asynchronously and the returned promise will
    * be resolved when the operation completes.
    *
-   * @param  {Record|Record[]|QueryResult} _records - the records to delete
+   * @param  {Record|Record[]|QueryResult} attr - the records to delete
    * @return {Promise} promise
    */
-  async delete(_records) {
-    let records = _records;
-    let isQueryResult = records instanceof QueryResult;
-    if (!_.isArray(records) && !isQueryResult) {
-      records = [records];
-    }
-
-    let ids = _.map(records, perRecord => perRecord.id);
-    let payload = {
+  async delete(attr) {
```

Comment:
maybe call the `attr` an `obj` like [here](https://github.com/SkygearIO/skygear-SDK-JS/pull/445/files#diff-c5582aa5d855002f6f1592d1724ade87R507)